docs/builder_image.md

```diff
@@ -132,3 +133,19 @@ This is a STI sample builder image, to use it, install
 https://github.com/openshift/source-to-image
 EOF
 ```
+
+## test/run
+
+The `test/run` script is for you, as the builder image author, to create a simple
+process to checks if the image is working correctly. The proposed flow of that process
+should be following:
+
+1. Build the image.
+1. Run the image to verify `usage` script.
+1. Run `sti build` to verify `assemble` script.
+1. (optional) Run `sti build` once more to verify `save-artifacts` script and
+   `assemble`'s restore artifacts functionality.
+1. Run the image to verify the test application is working.
+
+**NOTE** The suggested place to put your test application is `test/test-app` in your image repository,
```

Comment:
"to put your test application which should be built by your test/run script is..."